feature-group-definitions/baseline-source.yml

```
@@ -0,0 +1,8 @@
+name: baseline-source
+description: The `baseline-source` CSS property controls how inline-level boxes with multiple lines of text are aligned with the surrounding text. By default, which baseline is used depends on the `display` property value.
```

Mostly me just trying to work in the word "typographic" to give this a little more context.

```suggestion
description: The `baseline-source` CSS property controls how inline-level boxes with multiple lines of text are aligned with the surrounding text. By default, the `display` property value sets the typographic baseline.
```

---

I think working in "typographic" is good, but "sets" feels wrong here. The choice depends on `display`, but that's not at all obvious and more of an accident than by design. It's good if the unfortunateness of this comes through in the description as much as possible. Minimally:

```suggestion
description: The `baseline-source` CSS property controls how inline-level boxes with multiple lines of text are aligned with the surrounding text. By default, which typographic baseline is used depends on the `display` property value.
```

---

OK, cool—I do not want to imply intention that does not exist. Your version works for me.